modules/script/script-impl/src/test/resources/myapplication/require-test.js

```diff
@@ -0,0 +1,29 @@
+const innerLib = require('inner-script');
+
+console.log('I am a require function');
+
+log.info("Print logs with two arguments: %s and %s ", 123.09, "456");
+log.info('info message', new Error('cause'));
+
+exports.get = function(req) {
+    innerLib.myFunction();
+
+    console.log('GET function is called');
+
+    console.log(JSON.stringify(app, null, 2));
+    console.log(JSON.stringify(xxx, null, 2));
+
+    var bean = __.newBean('com.enonic.xp.script.graal.GraalScriptBean');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
    var bean = __.newBean("com.enonic.xp.script.graal.GraalScriptBean");
```